ui/pages/extensions.js

```diff
@@ -201,11 +211,16 @@ const Extensions = ({ classes, toggleCatalogContent,  enqueueSnackbar, closeSnac
 }
 
 const mapStateToProps = state => ({
-  catalogVisibility : state.get('catalogVisibility')
+  catalogVisibility : state.get('catalogVisibility'),
+  capabilitiesRegistry : state.get('capabilitiesRegistry')
 })
 
 const mapDispatchToProps = dispatch => ({
   toggleCatalogContent : bindActionCreators(toggleCatalogContent, dispatch)
 })
 
+export const extensions = {
```

Comment:
Define this constant in Enum.js (where all other constants are defined)

Reply:
okay.